style.css

```diff
@@ -562,3 +595,222 @@ button:disabled {
   background: #f4f1f0;
   color: #5b5957;
 }
+
+@media screen and (min-width: 1024px) {
+  .menu {
+    display: none;
+  }
+
+  .menu-list {
+    display: flex;
+    justify-content: space-around;
+    width: 293px;
+    align-items: center;
+    font-family: Inter, Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    color: #fff;
+  }
+
+  .menu-list img {
+    margin-top: 6px;
+  }
+
+  .intro {
+    background-image: url(./img/Illustration-header.png);
+    height: 870px;
+    flex-direction: row;
+    justify-content: flex-start;
+    align-items: center;
+  }
+
+  .intro-details p {
+    max-width: 566px;
+  }
+
+  .social-lg {
+    list-style: none;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    width: 20px;
+    margin-right: 87px;
+  }
+
+  .social {
+    display: none;
+  }
+
+  .title-header {
+    flex-direction: row;
+    width: 100%;
+    padding-bottom: 0;
+    margin-bottom: 0;
+  }
+
+  .title-header h2 {
+    display: flex;
+    align-items: flex-end;
+    margin-right: 24px;
+    width: 315px;
+  }
+
+  .title-border {
+    min-width: 794px;
+  }
+
+  .work {
+    flex-direction: row;
+    justify-content: center;
+    align-items: flex-start;
+    margin-top: 0;
+  }
+
+  .mobile-img {
+    display: none;
+  }
+
+  .desk-img {
+    max-width: 100%;
+    display: flex;
+  }
+
+  .work section {
+    max-width: 684px;
+  }
+
+  .recent-works {
+    margin-left: 141px;
+    margin-right: 143px;
+  }
+
+  .d-flex {
+    flex-direction: row;
+    justify-content: center;
+    flex-wrap: wrap;
+  }
+
+  .work-2 {
+    display: none;
+  }
+
+  .d-work {
+    background-size: cover;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  .proj-work {
+    display: none;
+  }
+
+  .proj-work-1 {
+    width: 327px;
+    background-image: url(./img/Img-Placeholder-1.png);
+    background-repeat: no-repeat;
+    margin: 24px;
+    height: 435px;
+    display: flex;
+    align-items: flex-end;
+  }
+
+  .proj-btn {
+    margin: 0;
+  }
+
+  .proj-work-2 {
```

Comment:
- [ ] Between 1024px and around 1300px, your projects section is not responsive. Look into this. 😄 

![Screenshot from 2021-08-20 04-05-00](https://user-images.githubusercontent.com/65501535/130163008-f3c80489-c7f7-4b94-a2fa-209dac3fdf2b.png)